dfttk/magmom_dict.yaml

```diff
@@ -0,0 +1,44 @@
+Co: 5
```

Comment:
I modified the vasp_input module so that if an element is not found in the MAGMOM_DICT, then it will be set to 0. You can specify any desired MAGMOM by supplying the other_settings argument for the moment.